edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudProxy.cs

```diff
@@ -152,12 +153,18 @@ public async Task SendMessageAsync(IMessage inputMessage)
                 string outputRoute = inputMessage.GetOutput();
                 using (Metrics.TimeMessageSend(this.clientId, outputRoute))
                 {
+                    var tokenSource = new CancellationTokenSource(this.sdkWaitTime);
```

Comment:
I thought this doesn't really do anything?

Reply:
Yes, this doesnt do anything and should be removed. Passing the cancellationtoken to the client did not work, so we instead implemented a task extension